cloud-sql/mysql/sqlalchemy/deployment.yaml

```diff
@@ -67,6 +75,10 @@ spec:
           # following flag to have the proxy connect over private IP
           # - "--private-ip"
 
+          # If you are not connecting with Automatic IAM, you can delete
+          # the following flag.
+          - "--auto-iam-authn"

```

Comment:
Just confirming you want this to be on by default? It conflicts with the advice above that using Automatic IAM is recommended but it's on by default here